bin/compile

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+cp -Rf * $1
```

Comment:
It copies the files of the current directory to the application path (which will be persisted).
The application path is passed to this script as the first argument, it works in the same way for Cloud Foundry too.
The final stage, which is used to actually boot the application, contains any files contained in the application path.